module_bindings/dripline_core/run_simple_service_pybind.hh

```diff
@@ -0,0 +1,12 @@
+#include "run_simple_service.hh"
+#include "pybind11/pybind11.h"
+
+namespace dripline_pybind {
+
+  void export_run_simple_service(pybind11::module& mod){
+    pybind11::class_<dripline::run_simple_service, std::shared_ptr<dripline::run_simple_service> >(mod, "run_simple_service")
+      .def(pybind11::init<const scarab::param_node&>())
```

Comment:
spaces for `()` and `<>`, can skip doc string on `__init__` if it takes no arguments